gatsby/pages.rst

```diff
@@ -1,7 +1,8 @@
 Pages
 =====
 
-Pages are the basic components of a GatsbyJS site.
+Pages can be considered similar to HTML documents in a basic website as they behave that way, but here, internally
+they are React components instead with way more functionality and can even be dynamically generated.
 
 If we see the file structure of our `hello-world` project, we can see that there is a `pages` folder with some js files:
 
```

Comment:
JavaScript files, if you want to use `js` make sure to explain to reference to a glossary, do not expect that people know for what it stands for :)

Reply:
Should we add "js" to the [Preferred Word Choice table](https://docs.plone.org/about/contributing/word_choice.html#preferred-word-choice) as something _not_ to use?

Reply:
Personally I prefer not to use it like it is used here, there is of course nothing wrong with having a file called *files.js*, but for the sake of readability for the rest I would prefer JavaScript :).

IMHO even with a glossary this can be confusing, at least when you not work daily with code and/or you are not a experienced developer person :)